CHANGELOG.md

```diff
@@ -1,8 +1,45 @@
 # Changelog
 
-## [v1.8.0-dev] - Unreleased
+## [v1.8.2-dev] - Unrealeased
 
 ### Added
+
+- Lateness can now be limited using the notion of `maximum lateness` as a field of each timewindow [#303](https://github.com/Mapotempo/optimizer-api/pull/303)
```

Comment:
Timewindow violation ("lateness") can now be limited using the `:maximum_lateness` field of timewindows. By default, `:maximum_lateness` is set to be 100% of the timewindow -- i.e., `:end - :start`. The default value can be overridden via the environment variable `OPTIM_DEFAULT_MAX_LATENESS_RATIO`.